src/insights.ts

```diff
@@ -55,20 +58,21 @@ class AlgoliaInsights {
     }
 
     const initAction = flushedActions.shift();
+    // eslint-disable-next-line prefer-spread
```

Comment:
If we have a lint rule which prefers spreads… just use a spread.

```
this.init(...(initAction?.args ?? []))
```

Having written that out, I will admit that it's uglier. It saves the eslint-disable line tho!

Reply:
It does save an eslint-disable, but also creates a bunch of new type errors.

Though I think we can make a couple of quick changes to this to make it a little more explicit.